src/devhub/page/community/index.jsx

```diff
@@ -290,7 +295,7 @@ return (
                   aria-current={tab === title && "page"}
```

Comment:
Can we normalize the tab title in the href when linked

Reply:
oh yes, sure, I missed that